src/ansys/geometry/core/designer/geometry_commands.py

```diff
@@ -79,3 +79,33 @@ def chamfer(
         )
 
         return result.success
+
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def fillet(
+        self, selection: Union["Edge", List["Edge"], "Face", List["Face"]], radius: Real
+    ) -> bool:
+        """Create a fillet on an edge or adjust the fillet of a face.
+
+        Parameters
+        ----------
+        selection : Edge | list[Edge] | Face | list[Face]
+            One or more edges or faces to act on.
+        radius : Real
+            Fillet radius.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+        selection: list[Edge | Face] = selection if isinstance(selection, list) else [selection]
+
```

Comment:
Verify all elements in the selection are of type Edge or Face. As well as the radius. Use the checks here: radius (https://github.com/ansys/pyansys-geometry/blob/9ac5481833c28702365a1e0b00819d6faa620f91/src/ansys/geometry/core/misc/checks.py#L93) and Face or Edge in list (https://github.com/ansys/pyansys-geometry/blob/9ac5481833c28702365a1e0b00819d6faa620f91/src/ansys/geometry/core/misc/checks.py#L279)